src/helper/GetStars.tsx

```diff
@@ -0,0 +1,5 @@
+const getStars = (numberOfStars: number) => {
+    return Array.from(Array(numberOfStars).keys());
+}
+
+export default getStars;
```

Comment:
Ovaj file ne bih nazvala ovako, radije ga nazovi `helpers.tsx` (i definitivno ne velikim početnim slovom jer će to davati dojam da se radi o komponenti, a radi se samo o pomoćnim funkcijama).
`getStars` ne mora biti jedina pomoćna funkcija, kasnije ovdje možeš dodati i druge pomoćne funkcije koje se koriste na više mjesta u kodu.
Tako da ti je moj savjet ovdje staviti sljedeće:
`export const getStars = (numberOfStars: number) => {`
Onda možeš pobrisati liniju `export default getStars;`